FSharpLu.Json/Compact.fs

```diff
@@ -28,6 +28,20 @@ module private ConverterHelpers =
     
     let getUnionCasesMemorised = memorise FSharpType.GetUnionCases
 
+    let constructUnionCaseMemorised = memorise FSharpValue.PreComputeUnionConstructor
+
+    let getUnionCaseProperyInfoFieldsMemorised = memorise (fun (case: UnionCaseInfo) -> case.GetFields())
+
+    let findNoFieldsMatchingUnionCaseByNameAndType  = 
```

Comment:
Naming consistency: `findNoFieldsMatchingUnionCaseByNameAndType` does not end with `Memorised` like the two let bindings above. I recommend moving everything under `Memorised` module and removing the suffix everywhere.